src/main/java/org/zendesk/client/v2/Zendesk.java

```diff
@@ -2333,6 +2340,14 @@ public Iterable<Translation> getCategoryTranslations(Long categoryId) {
                 tmpl("/help_center/categories/{categoryId}/translations.json").set("categoryId", categoryId),
                 handleList(Translation.class, "translations"));
     }
+
+    public Translation getCategoryTranslation(long categoryId, String locale) {
```

Comment:
I don't see any endpoint to show translations for Category, am I missing something?